lib/task/tools/renameSlugTask.class.php

```diff
@@ -19,6 +19,8 @@
 
 class renameSlugTask extends arBaseTask
 {
+    protected static $globalErrors = [];
```

Comment:
Small nitpick but maybe consider naming this array `failedSlugs` or something similar so that it describe what it contains more clearly.